src/content/docs/journey-app-slow/root-causes.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Prepare to triage your application
```

Comment:
Something about this title feels a bit arms-length to me. Like the reader won't be taking any action here, just preparing to do triaging and then taking action.